src/components/Results/Results.tsx

```diff
@@ -33,21 +35,81 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
   const results = useRecoilValue(resultsState);
   const detectedIssues = fhirpath.evaluate(results, 'Bundle.entry.resource.DetectedIssue');
 
-  return (
-    <Grid container xs={12} direction="column" spacing={2}>
-      <h2>Results:</h2>
-      <Grid container item xs={12} direction="row" justify="center" alignItems="center">
-        {outputType === 'measureReports' &&
-          calculationOptions.reportType === 'individual' &&
-          fhirpath.evaluate(results, 'MeasureReport.group').map((group: R4.IMeasureReport_Group) => {
-            const id = results ? fhirpath.evaluate(results, 'MeasureReport.subject.reference')[0].split('/') : '';
+  const style = {
+    minWidth: '25%'
+  };
+
+  const [value, setValue] = React.useState('0');
```

Comment:
Mind renaming this variable to indicate its purpose more? Maybe like `selectedTab` or something since that's what it represents

Reply:
also.... thoughts on using an enum here instead of a string indexing the tabs? Typescript gives us this ability, and it makes it a lot clearer what's going on with the tab switching. It'd look something like this:

``` Typescript
enum TabValues {
  JSON = '0',
  HTML = '1',
  TABULAR = '2',
  ACCORDION = '3'
}
```

Then, you would say initialize your state variable to JSON by default like this instead:

``` Typescript
const [value, setValue] = React.useState(TabValues.JSON);
```

You'd also want to use this enum in the actual tabs for indicating their values. This just allows us to actually name the tabs rather than use an index, and could help us in the future if we add new tabs or re-order them